src/test/java/org/candlepin/subscriptions/subscription/SubscriptionSyncControllerTest.java

```diff
@@ -363,7 +377,7 @@ void findsSubscriptionId() {
   }
 
   @Test
-  void memoizesSubscriptionId() {
+  void memorizesSubscriptionId() {
```

Comment:
I think this naming was actually correct the first time.

https://en.wikipedia.org/wiki/Memoization#:~:text=In%20computing%2C%20memoization%20or%20memoisation,the%20same%20inputs%20occur%20again.

Reply:
Ahh didn't know this was a word! I think intellij gave me a squiggly cuz it doesn't know either.